### PR TITLE
Review getting started document

### PR DESCRIPTION
**Pull request by Preeticp**
## What is the purpose of this change? What does it change?
Reviewed and modified getting started document.

### CODE REVIEW

**Comment by Preeticp** [comment]
@cdrage request you to review the PR please.

File: docs/getting-started.md [hunk 1]
```diff
@@ -1,281 +1,141 @@
...
+### Using Service Catalog with odo
 
-### Service Catalog
+If you use `minishift` you need to use minishift version 1.22 or above.
 
 In order to use the Service Catalog it must be enabled within your OpenShift cluster.
 
-Requirements:
-  - `minishift` version 1.22+
+ 1. Start an OpenShift cluster, version 3.10.0 and above
```
Comment (cdrage) [inline docs/getting-started.md]: i think `3.10` and above is fine

File: docs/getting-started.md [hunk 2]
```diff
@@ -1,281 +1,141 @@
...
+ 2. Enable the Service Catalog:
+ ```sh
+  MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--enable=*,service-catalog,automation-service-broker"
+ ```
 
-If you are using `minishift` you'll need to start an OpenShift cluster with version 3.10.0+ and Service Catalog explicitly enabled.
+3. After you enable or start `minishift` use:
+ * `odo catalog list services` to list the services
+  * `odo service <verb> <servicename>` to list service catalog related operations
```
Comment (cdrage) [inline docs/getting-started.md]: odd spacing here...

File: docs/getting-started.md [hunk 3]
```diff
@@ -1,281 +1,141 @@
...
 
-```sh
-# Deploy minishift
-MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--enable=*,service-catalog,automation-service-broker"
-```
-
-After you've enabled / started `minishift`, you'll be able to list the services via `odo catalog list services` and service catalog related operations via `odo service <verb> <servicename>`.
-
-## Adding a Custom Builder
+### Adding a Custom Builder
```
Comment (cdrage) [inline docs/getting-started.md]: Should Custom Builder be capitalized?

File: docs/getting-started.md [hunk 4]
```diff
@@ -1,281 +1,141 @@
...
 
-**This section assumes that the `oc` binary has been [installed](https://docs.openshift.org/latest/cli_reference/get_started_cli.html#installing-the-cli) and is present on the $PATH**
+OpenShift enables you to add a [custom image](https://docs.openshift.com/container-platform/3.7/creating_images/custom.html) to bridge the gap in the creation of custom images. A custom builder image usually includes the base image of [openshift/origin-custom-docker-builder](https://hub.docker.com/r/openshift/origin-custom-docker-builder/).
```
Comment (cdrage) [inline docs/getting-started.md]: bridge the gap between the creation

File: docs/getting-started.md [hunk 5]
```diff
@@ -1,281 +1,141 @@
...
 
-OpenShift includes the ability to add a [custom image](https://docs.openshift.com/container-platform/3.7/creating_images/custom.html) to bridge the gap in the creation of custom images.
+The following example demonstrates the successful import and use of [redhat-openjdk-18](registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift) image:
 
-A custom builder image usually includes the base image of [openshift/origin-custom-docker-builder](https://hub.docker.com/r/openshift/origin-custom-docker-builder/).
+**Prerequisites:**
+oc binary is [installed](https://docs.openshift.org/latest/cli_reference/get_started_cli.html#installing-the-cli) and present on the `$PATH`
```
Comment (cdrage) [inline docs/getting-started.md]: `oc`

**Comment by Preeticp** [comment]
@cdrage I have addressed all your comments except the TOC part. I would suggest let's revisit this after we have done a round of review of all docs, so that we can have a consistent feel to the whole doc set. WDYT?
Please merge if you feel it's alright, I will proceed with the Installation doc.

**Comment by amitkrout** [comment]
@Preeticp Some prow fixes went in, so can debase please.

**Comment by Preeticp** [comment]
@amitkrout I have rebased, fixed the merge conflict. Reviewed and modified the new section which created the merge conflict. 
Please merge.

**Pull request merged**